js/main.js

```diff
@@ -0,0 +1,15 @@
+const generateRandomNumber = (min, max) => {
+  const range = max - min;
+  if (range > 0) {
+    return Math.random() * (range + 1) + min;
+  }
+  return min >= 0 ? min : 0;
+
+};
```

Comment:
```suggestion
  }
  
  return min >= 0 ? min : 0;
};
```

https://learn.javascript.ru/coding-style